dst.go

```diff
@@ -258,10 +258,11 @@ type (
 
 	// A FuncType node represents a function type.
 	FuncType struct {
-		Func    bool
-		Params  *FieldList // (incoming) parameters; non-nil
-		Results *FieldList // (outgoing) results; or nil
-		Decs    FuncTypeDecorations
+		Func       bool
+		TypeParams *FieldList
```

Comment:
Let's copy the comment from [ast.go](https://github.com/golang/go/blob/master/src/go/ast/ast.go#L450)